openapi/kas-fleet-manager.yaml

```diff
@@ -890,6 +890,8 @@ components:
               type: string
             instance_type:
               type: string
+            display_name:
+              type: string
```

Comment:
The suggestion in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/868#discussion_r832277564 of changing the `display_name` to `instance_type_name` or `instance_type_display_name` may have to be applied here if accepted.